model/Core/Classes/DictionaryEntry.md

```diff
@@ -0,0 +1,27 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DictionaryEntry
+
+## Summary
+
+A key with an associated value.
+
+## Description
+
+The class used for implementing a generic string mapping (also known as associative arrays, dictionaries, and hash maps) in SPDX.  Each DictionaryEntry contains a key-value pair which maps the key to its associated value.  To implement a dictionary, this class is to be used in a collection with unique keys.
```

Comment:
```suggestion
The class used for implementing a generic string mapping (also known as associative array, dictionary, or hash map) in SPDX.  Each DictionaryEntry contains a key-value pair which maps the key to its associated value.  To implement a dictionary, this class is to be used in a collection with unique keys.
```
As "a generic string mapping" is singular, the following enumeration should be, too. The final "and" only makes sense to me if we know the list to be exhaustive. If that is the case, then ignore my "or" suggestion.